patches/node/.patches

```diff
@@ -42,3 +42,4 @@ fix_preserve_proper_method_names_as-is_in_error_stack.patch
 macos_avoid_posix_spawnp_cwd_bug_3597.patch
 src_update_importmoduledynamically.patch
 fix_add_v8_enable_reverse_jsargs_defines_in_common_gypi.patch
+sandbox_disallow_arraybuffers_outside_the_vm_cage.patch
```

Comment:
Do not merge with this patch without deciding a way forward for V8 sandbox with the `upgrades-wg`

Reply:
We ended up deciding to disable v8 sandboxed pointers